.github/workflows/build.yml

```diff
@@ -224,20 +224,48 @@ jobs:
     - uses: actions/upload-artifact@v2
       with:
         path: wheelhouse/*.whl
-    - name: Build a source tarball
-      run: >-
-        python -m
-        build
-        --sdist
-        --outdir wheelhouse/
-    - if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/main' }}
-      name: Publish distribution 📦 to Test PyPI
-      uses: pypa/gh-action-pypi-publish@release/v1
-      with:
-        user: __token__
-        password: ${{ secrets.TEST_PYPI_API_TOKEN }}
-        repository_url: https://test.pypi.org/legacy/
-        packages_dir: wheelhouse/
+        name: wheels
+
+  pypi:
+    if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/main' }}
+    name: Upload wheels to pypi
+    runs-on: ubuntu-latest
+    needs: build
+    steps:
+      - uses: actions/checkout@v1
+      - name: Download binary wheels
+        id: download
+        uses: actions/download-artifact@v2
+        with:
+          name: wheels
+          path: wheelhouse
+      - uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
```

Comment:
This seems wrong, how did this step pass on the PR? `matrix.python-version` shouldn't be set here.